### PR TITLE
fix(statutory): add watch live button during Agora

### PR DESCRIPTION
**Pull request by WikiRik**

### CODE REVIEW

**Comment by WikiRik** [comment]
build is not passing due to temporary issues with CircleCI

File: src/views/statutory/Single.vue [hunk 1]
```diff
@@ -11,6 +11,13 @@
       </div>
       <div class="tile is-parent">
         <article class="tile is-child is-info">
+          <div class="field is-grouped" v-if="duringAgora">
+            <a href="https://aegeeagora-streaming.westeurope.cloudapp.azure.com/" class="button is-danger is-fullwidth">
```
Comment (serge1peshcoff) [inline src/views/statutory/Single.vue:15]: would it be static or would it change every Agora?

**Review by serge1peshcoff — APPROVED** [review]

File: src/views/statutory/Single.vue (hunk 1 above)
Comment (WikiRik) [reply src/views/statutory/Single.vue:15]: It's static, although we might want to get a nicer link for next Agora

**Pull request merged**